analysis/tests/src/expected/CompletionJsxProps.res.txt

```diff
@@ -174,17 +180,18 @@ posCursor:[18:29] posNoWhite:[18:28] Found expr:[18:12->18:32]
 JSX <div:[18:12->18:15] onMouseEnter[18:16->18:28]=...__ghost__[0:-1->0:-1]> _children:18:30
 Completable: Cexpression CJsxPropValue [div] onMouseEnter
 Package opens Pervasives.JsxModules.place holder
+Resolved opens 1 pervasives.res
 ContextPath CJsxPropValue [div] onMouseEnter
 Path ReactDOM.domProps
-Path JsxDOM.domProps
+Path Pervasives.JsxDOM.domProps
 [{
-    "label": "event => {}",
+    "label": "v => {}",
     "kind": 12,
     "tags": [],
-    "detail": "JsxEvent.Mouse.t => unit",
+    "detail": "JsxEventC.Mouse.t => unit",
```

Comment:
That's the curried one.
We need 2 copies of pervasives. One for curried, one for uncurried.